examples/set-project-id-advanced/.expected/diff.patch

```diff
@@ -29,7 +29,7 @@ index c96b2cd..2643351 100644
    name: subpkg
```

Comment:
Looking at the output from the CI:
```
        Diff of Diff
        ===
        --- /tmp/kpt-e2e-diff-087313666/expected	2021-08-30 20:36:14.544060022 +0000
        +++ /tmp/kpt-e2e-diff-087313666/actual	2021-08-30 20:36:14.544060022 +0000
        @@ -20,7 +20,7 @@
            storage-class: standard
         +  project-id: foo
         diff --git a/subpkg/Kptfile b/subpkg/Kptfile
        -index c96b2cd..2643351 100644
        +index c96b2cd..52b8640 100644
         --- a/subpkg/Kptfile
         +++ b/subpkg/Kptfile
         @@ -2,3 +2,8 @@ apiVersion: kpt.dev/v1
```

Line 23 `index c96b2cd..2643351 100644` needs to be updated to `index c96b2cd..52b8640 100644`.

Reply:
Updated it! 👍